test/es-module/test-esm-import-json-named-export.mjs

```diff
@@ -0,0 +1,22 @@
+import '../common/index.mjs';
+import { ok } from 'assert';
+import { spawn } from 'child_process';
+import { execPath } from 'process';
+
+const child = spawn(execPath, [
+  '--experimental-json-modules',
+  './test/fixtures/es-modules/import-json-named-export.mjs',
```

Comment:
Let's avoid having the test rely on the cwd. Same in subsequent files.
```suggestion
  fixturesPath('es-modules', 'import-json-named-export.mjs'),
```

Reply:
That doesn’t work, I guess because it’s using the CommonJS loader for resolution:

```
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '/Users/Geoffrey/Sites/node/file:/Users/Geoffrey/Sites/node/test/fixtures/es-modules/import-json-named-export.mjs'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
```

Is there a reason we can’t rely on the cwd? All the CI tests pass.